lib/util.js

```diff
@@ -219,7 +219,7 @@ function setErrorHandler(app) {
 /**
  * Creates a new router with some default options.
  *
- * @param {Object} opts additional options to pass to express.Router()
+ * @param {Object=} opts additional options to pass to express.Router()
```

Comment:
Hm.. I was generally following the JSDoc syntax, not the closure compiler syntax (http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values), but I think we don't have any official policy on this. However in every other place it's `[opts]` right now. Let's keep the JSDoc syntax?